test/fail_compilation/fail_shared.d

```diff
@@ -0,0 +1,24 @@
+/*
+REQUIRED_ARGS:-preview=restrictiveshared
+TEST_OUTPUT:
+---
+fail_compilation/fail_shared.d(15): Error: Trying to Access shared state `x`
+fail_compilation/fail_shared.d(16): Error: Trying to Access shared state `x`
+fail_compilation/fail_shared.d(17): Error: Trying to Access shared state `x`
+fail_compilation/fail_shared.d(22): Error: Trying to Access shared state `x`
+---
+*/
+
+
+int f1(shared int x)
+{
+    x += 7; // Error: Trying to accsess shared state x
```

Comment:
I think the error should say "Invalid access to shared data x"
'Trying' feels like an intent rather than an error.

Reply:
Spelling errors